app/views/play/CampaignView.coffee

```diff
@@ -1137,16 +1131,6 @@ module.exports = class CampaignView extends RootView
     if what is 'codeplay-ads'
       return !me.finishedAnyLevels() && serverConfig.showCodePlayAds && !features.noAds && me.get('role') isnt 'student'
 
-    if what in ['status-line']
-      return true unless isStudent
-      return false unless @classroom?
-      return @classroom.getSetting('gems') || @classroom.getSetting('xp')
-
-    if what in ['gems']
-      return true unless isStudent
-      return false unless @classroom?
-      return @classroom.getSetting('gems')
-
```

Comment:
Is the classroom gems setting no longer needed? If so then it should be removed from the admin interface for it.